test/functional/toolchains/qbs/test_qbsdeps.py

```diff
@@ -111,6 +111,7 @@ class Recipe(ConanFile):
             exports_sources = "*.cpp", "*.h", "*.qbs"
             settings = "os", "compiler", "build_type", "arch"
             generators = "QbsDeps"
+            generators_folder = "generators"
```

Comment:
This is not necessary/correct, the ``generators_folder`` is defined by the ``layout()`` method with ``self.folders.generators = xxxx``. It is important that it should also be used by the ``QbsDeps`` generator to actually write the files there, I am not sure this is happening now?

Reply:
Well, the test passes, so I assume it uses the "generators" dir

Reply:
yes, that is right, the ``generate()`` uses ``save(self._conanfile, os.path.join('conan-qbs-deps', file_name)`` which is a relative path, and it runs in the ``generators_folder`` by default. Looks good then.